code/game/objects/items/weapons/storage/book.dm

```diff
@@ -156,3 +160,41 @@ GLOBAL_LIST_INIT(bibleitemstates, list("bible", "koran", "scrapbook", "bible",
 
 /obj/item/weapon/storage/book/bible/booze/PopulateContents()
 	new /obj/item/weapon/reagent_containers/food/drinks/bottle/whiskey(src)
+
+/obj/item/weapon/storage/book/bible/syndicate
+	icon_state ="ebook"
+	deity_name = "The Syndicate"
+	throw_speed = 2
+	throwforce = 18
+	throw_range = 7
+	force = 18
+	hitsound = 'sound/weapons/sear.ogg'
+	damtype = BURN
+	name = "Syndicate Tome"
+	attack_verb = list("attacked", "burned", "blessed", "damned", "scorched")
+	var/ordained = FALSE
```

Comment:
I think `used` would be a more fitting var. I'm trying not to bikeshed here but it will be consistent with other objects and as such can be more easily found by admins.